src/constants.rs

```diff
@@ -97,3 +97,5 @@ pub const UNWRAP_EMOJI: Emoji<'_, '_> = Emoji("🔩 ", "");
 pub const MAX_FREEZE_DAYS: u8 = 31;
 
 pub const COMPUTE_UNITS: u32 = 400_000;
+
+pub const PRIORITY_FEE: u64 = 20_000;
```

Comment:
Is there a specific reason you chose this value? Would it make more sense to parameterize it so that the user could reduce or increase it on demand?

Reply:
It's trial and error. I picked up this number from https://solana.com/developers/guides/advanced/how-to-use-priority-fees. But honestly, the numbers are all over the place (in here they use 300 for CU).

I did put it in the client_config at first, but dropped it when I saw the constant here. Is that where you'd prefer to parameterize it? I agree it's not great to have it as a hard-coded constant (though it's a handy default).

Reply:
The priority fee number you might need depends on the other transactions that are competing with the tx to get into the block - therefore it might be helpful to have a parameter for the user to set for the PRIORITY_FEE value.
Having a value greater 0 as default still might make sense in addition :)

Reply:
Definitely agree. Just to be 100% clear, it would be added somewhere here? https://github.com/metaplex-foundation/sugar/blob/886c6f063b40a0f3504a23711268d11d85d226a8/src/config/data.rs#L30

It would be necessary to inject the config into a few methods that don't have it presently, but that's no biggie.

Reply:
I personally would not add it to the config (which is basically read from the config.json), but rather have a Argument like we have to override solana rpc settings for example.
You would have to add the --priority-fees argument in the src/cli/mod.rs file and then add it to all the functions where you added the priority fees.

You could have a look at this PR which added an additional argument, too, so you might be able to reuse some of these changes. https://github.com/metaplex-foundation/sugar/commit/56f0a4cf5691f958c087658e6dce9bae51b0e8cf 

Does this help? Please let me know in  case you need any more assistance - i would love to see it merged :)

Reply:
Added it as a parameter 
<img width="469" alt="image" src="https://github.com/metaplex-foundation/sugar/assets/141178592/5b232402-292c-40b5-b32d-29143bfa1434">

For the most part I either pass in priority_fee as a single arg or pass along the whole args object. Clippy warned about too many arguments, so I had to override.

I haven't tested all functions, but I did do a full launch and guard add + update. Some example TXs:

default:
https://solana.fm/tx/juF7ngQHrGwx3K9mM9nfjsNeCUbTPtd6tpd3F5gDUNqKAeV48NZfariqpnQMyTuRhLvmFQe7rBmSDrNcWJZGzdi

overridden to 1200 https://solana.fm/tx/6Z2sAca8VZ6TtHtnMrdHqVxAmRdEqUNbDpTjrZXyHfBbkLzjAdGvQd7cb3dTAcjX4cWuQY6KMc8AFNoXTmrNV4E

overridden to 0:
https://solana.fm/tx/5fNP5ZPNzSWWyi1boHywfZRpPGGwCUuWc5i2QPmb5jkTc6BdEZh1of8wNXb6UYWcvSn7FztkBLowV9pVd8g8SSEU

Reply:
Also tested a full 5K launch. Granted, still devnet :) 
<img width="1024" alt="image" src="https://github.com/metaplex-foundation/sugar/assets/141178592/997d2e2a-3fb1-4eb6-96ce-d8d7d662aa4a">

Reply:
Thank you!